requirements/static/ci/py3.5/windows-crypto.txt

```diff
@@ -0,0 +1,9 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile -o requirements/static/ci/py3.5/windows-crypto.txt -v requirements/crypto.txt requirements/static/ci/crypto.in
+#
+m2crypto==0.35.2
```

Comment:
I don't think there are wheels available for 35.2 on Windows...

Reply:
Nope. Pretty sure M2Crypto won't build off the source with pip on Windows. That's why I've been bundling pycryptodome on Windows packages.